.github/workflows/package-release.yml

```diff
@@ -35,11 +35,13 @@ jobs:
         run: yarn test
 
       - name: Package library
-        run: yarn pack
+        # %s-%v.tgz => @trussworks-react-uswds-v*.*.*.tgz
+        # https://v3.yarnpkg.com/cli/pack
+        run: yarn pack --out %s-%v.tgz
 
       - name: Upload package
         uses: actions/upload-artifact@v4
         with:
-          path: trussworks-react-uswds-v*.*.*.tgz
+          path: @trussworks-react-uswds-v*.*.*.tgz
```

Comment:
Hmm, this causes problems:
<img width="479" alt="image" src="https://github.com/user-attachments/assets/055a0099-a8c7-4e3a-bfd0-f8e0522e6a28" />

Reply:
Fixed to use the old file name:
<img width="246" alt="image" src="https://github.com/user-attachments/assets/a69bf818-8560-4b56-bea6-45232c1f69c3" />

Reply:
Side note, intelliJ is the coolest:
<img width="745" alt="image" src="https://github.com/user-attachments/assets/e98a1a7f-b138-4c3d-b769-bbcbc1e12d19" />